src/inspector/view/qml/MuseScore/Inspector/notation/notes/BeamSettings.qml

```diff
@@ -170,9 +167,54 @@ FocusableItem {
                 }
             }
 
-            SeparatorLine {
-                anchors.margins: -12
-                visible: featheringControlsColumn.visible
+            DirectionSection {
+                id: beamDirection
+                visible: root.model ? !(root.model.iscrossStaffMoveAvailable) : true
+
+                titleText: qsTrc("inspector", "Beam direction")
+                propertyItem: root.model ? root.model.stemDirection : null
+
+                navigationPanel: root.navigationPanel
+                navigationRowStart: featheringRightSection.navigationRowEnd + 1
+            }
+
+            InspectorPropertyView {
+                id: crossStaffMove
+                visible: root.model ? root.model.iscrossStaffMoveAvailable : false
+
+                navigationName: "Move cross-staff beam"
+                navigationPanel: root.navigation.panel
+                navigationRowStart: beamDirection.navigationRowEnd + 1
+
+                titleText: qsTrc("inspector", "Move cross-staff beam")
+                propertyItem: root.model ? root.model.crossStaffMove : null
+
+                Row {
+                    spacing: 4
+                    width: parent.width
+
+                    FlatRadioButton {
+                        id: crossBeamUp
+                        width: 0.5 * parent.width - 2
+                        iconCode: IconCode.ARROW_UP
+                        checked: root.model ? root.model.crossStaffMove.value < 0 : false
+                        onClicked: root.model.crossStaffMove.value -= 1
```

Comment:
Just a code style thing that we usually do to make it easily visible that this is a signal handler rather than a property:
```suggestion
                        onClicked: {
                            root.model.crossStaffMove.value -= 1
                         }
```
(same applies to `crossBeamDown`)